dali/operators/util/get_property.cc

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/util/get_property.h"
+#include "dali/pipeline/data/types.h"
+
+namespace dali {
+
+DALI_SCHEMA(GetProperty)
+    .DocStr(
+        R"code(Returns a property of the tensor passed as an input.
+The type of the output will depend on the ``key`` of the requested property.)code")
```

Comment:
```suggestion
        R"code(Returns a data node property.

The output data type will depend on the property extracted. Supported properties are:
* ``"source_info"``: String-like (1D tensor of uint8) representing the source of the data node. Typically a file path associated with the data or other kind of identifier.
* ``"layout"``: String-like (1D tensor of uint8) representing the data layout. Each character names a dimension in the data shape. It can be used to understand the data layout (e.g. channel-first vs. channel-last).
)code")
```

Reply:
Note that the empty line between the first line and the rest is important for the operator table in the docs.

Reply:
Done